src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/Common.ps1

```diff
@@ -270,6 +270,22 @@ function Get-SqlServerKeyVaultKeyTestEnvironmentParameters ()
 
 <#
 .SYNOPSIS
+Gets the virtual network subnet id used in the virtual network rules tests
+#>
+function Get-VirtualNetworkSubnetId ($vnetName)
+{
+	$vnetSubscriptionId = "d513e2e9-97db-40f6-8d1a-ab3b340cc81a"
```

Comment:
The networking team has removed their vnet cmdlets because of a naming change. So I cannot use their cmdlets. I will create a new task to re-record tests once their cmdlets are in. Would that be ok?